apis/v1alpha1/cluster_fentry_program_types.go

```diff
@@ -44,11 +52,11 @@ const (
 	Dettach AttachTypeAttach = "Detach"
 )
 
-// ClFentryAttachInfo indicates that the Fentry program should be attached to
-// the function identified in ClFentryLoadInfo. The only valid value for Attach
-// is true.
 type ClFentryAttachInfo struct {
-	// +kubebuilder:validation:Enum=Attach;Dettach;
+	// mode is required and its presence indicates that the FEntry program should
+	// be attached. mode should be set to Attach. To detach the FEntry program,
+	// remove the link entry.
+	// +kubebuilder:validation:Enum=Attach;Detach;
 	Mode AttachTypeAttach `json:"mode"`
```

Comment:
```suggestion
	// +kubebuilder:validation:Enum=Attach;
	Mode AttachTypeAttach `json:"mode"`
```
The only option should be "Attach".  To detach, the link should be removed from the list.